projects/js-packages/components/components/boost-score-graph/index.tsx

```diff
@@ -22,15 +23,21 @@ export interface BoostScoreGraphProps {
 	startDate: number;
 	endDate: number;
 	title?: string;
-	isLoading?: boolean;
+	isPlaceholder?: boolean;
 }
 
+export type ScoreGraphAlignedData = [
+	number[], // timestamps
+	number[], // desktop_overall_score
+	number[] // mobile_overall_score
```

Comment:
Seems like we want this to be an object with keys? If there's an external dependency that requires an array with exactly 3 number entries, we can then do `Object.values()` when interacting with that component.

Reply:
This value itself is prepared for the third-party library. So, I don't think we should make it an object and then pass it through `Object.values` again.